README.md

```diff
@@ -6,22 +6,27 @@
 
 ---
 
-A collection of AWS resources ready to be used with [Humanitec](https://humanitec.com/).
+The AWS Resource Pack contains various reference implementation of connecting AWS with [Humanitec](https://humanitec.com/).
```

Comment:
I suggest being more specific here and name the product, i.e. "the Humanitec Platform Orchestrator". "Humanitec" is the name of the company and no longer synonymous with the Orchestrator now that we're multi-product.